lib/validate.js

```diff
@@ -31,3 +31,31 @@ exports.compare = function compare(oldSwagger, newSwagger, options) {
 
   return openApiDiff.compare(oldSwagger, newSwagger);
 };
+
+/**
+ * Wrapper method to compares old and new specifications.
+ *
+ * @param {string} oldSwagger Path to the old specification file.
+ *
+ * @param {string} oldTag Tag name used for autorest with the old specification file.
+ *
+ * @param {string} newSwagger Path to the new specification file.
+ *
+ * @param {string} newTagName Tag name used for autorest with the new specification file.
+ * 
+ * @param {object} options The configuration options.
+ *
+ * @param {boolean} [options.json] A boolean flag indicating whether output format of the messages is json.
+ * 
+ * @param {boolean} [options.matchApiVersion] A boolean flag indicating whether to consider api-version while comparing.
+ * 
+ */
+exports.compareTags = function (oldSwagger, oldTag, newSwagger, newTag, options) {
+  if (!options) options = {};
+
+  log.consoleLogLevel = options.consoleLogLevel || log.consoleLogLevel;
+  log.filepath = options.logFilepath || log.filepath;
+  let openApiDiff = new OpenApiDiff(options);
+
+  return openApiDiff.compare(oldSwagger, newSwagger, oldTag, newTag);
```

Comment:
since this will always be a Readme, maybe we can name it something like `oldReadme` `newReadme`

Reply:
Lets look at this in a subsequent checkin.